web/docs/contexts/bloom-filter.md

```diff
@@ -0,0 +1,64 @@
+# Bloom Filter
+
+A space-efficient data structure to represent large sets.
+
+## Synopsis
+
+```
+context create <name> bloom-filter --capacity <capacity> --fp-prob <probability>
+context update <name> --key <field>
+context delete <name>
+enrich <name> --field <field>
+lookup <name> --field <field>
+```
+
+## Description
+
+The `bloom-filter` context is a [Bloom
+filter](https://en.wikipedia.org/wiki/Bloom_filter) that stores large sets data
+in a compact way, at the cost of false positives during lookup.
+
+The Bloom filter has two tuning knobs:
+
+1. **Capacity**: the maximum number of items in the filter.
+2. **False-positive probability**: the chance of reporting an indicator not in
+   the filter.
+
+These two parameters dictate the space usage of the Bloom filter. Consult Thomas
+Hurst's [Bloom Filter Calculator](https://hur.st/bloomfilter/) for finding the
+optimal configuration for your use case.
+
+Bloom filter terminology commonly uses the following parameter abbreviations:
+
+| Parameter | Name | Description
+|:---------:|------|--------------
+| `n` | Capacity | The maximum number of unique elements that guarantee the configured false-positive probability
+| `m` | Size | The number of bits that the Bloom filter occupies
+| `p` | False positive probability | The probability of erroneously reporting an element to be in the set
+
+The Bloom filter implementation is a C++ rebuild of DCSO's
+[bloom](https://github.com/DCSO/bloom) library. It is binary-compatible and uses
+the exact same method for FNV1 hashing and parameter calculation, making it a
+drop-in replacement for `bloom` users.
+
+### `-n|--capacity <capacity>`
```

Comment:
There are no short-options for context parameters, so they also should not be in the docs. This also applies to the false-positive probability below.